components/ContractorInterestTable.tsx

```
@@ -0,0 +1,422 @@
+          return new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime();
+        case "oldest":
+          return new Date(a.createdAt).getTime() - new Date(b.createdAt).getTime();
+        case "name":
+          return a.name.localeCompare(b.name);
+        case "email_status":
+          // Sort by email status: failed, not sent, sent
+          const getStatusPriority = (contractor: InterestedContractor) => {
+            if (!contractor.welcomeEmailSent && contractor.welcomeEmailError) return 3; // Failed
+            if (!contractor.welcomeEmailSent) return 2; // Not sent
+            return 1; // Sent
+          };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)